pkgs/tools/wayland/wl-clipboard/default.nix

```diff
@@ -22,7 +23,7 @@ stdenv.mkDerivation rec {
 
   strictDeps = true;
   nativeBuildInputs = [ meson ninja pkg-config wayland-scanner ];
-  buildInputs = [ wayland wayland-protocols ];
+  buildInputs = [ wayland wayland-protocols xdg-utils ];
```

Comment:
This will not fix the issue since it is being used at run time and not search at build time: https://github.com/bugaevc/wl-clipboard/blob/6256adf6d936ec182e1daf6b7fd833a709a1ac5b/src/util/files.c#L173

You would either need to use `makeWrapper` to prepend to `PATH` or patch the linked file to refer to the full path to `xdg-mime`.

Reply:
I'll look further into this but from what I'm gathering, a simple `postInstall` stage like
```
  postInstall = ''
    wrapProgram $out/bin/wl-copy \
      --prefix PATH : ${xdg-utils}/bin/
  '';
``` 

should achieve this afaik